Classes/Domain/Repository/ImageFileReferenceRepository.php

```diff
@@ -32,45 +33,22 @@
  */
 class ImageFileReferenceRepository extends Repository
 {
-    /**
-     * @var ConnectionPool
-     */
-    private ConnectionPool $connectionPool;
-
-    /**
-     * @var Context
-     */
-    private Context $context;
+    private readonly ConnectionPool $connectionPool;
+    private readonly Context $context;
```

Comment:
If this is for TYPO3 v13 and therefore PHP >= 8.2, we should use

https://stitcher.io/blog/constructor-promotion-in-php-8